ferminet/train.py

```diff
@@ -44,6 +44,7 @@ def init_electrons(
     molecule: Sequence[system.Atom],
     electrons: Sequence[int],
     batch_size: int,
+    init_width: float,
```

Comment:
Add new argument to docstring please

Reply:
see [0e10b42](https://github.com/deepmind/ferminet/pull/42/commits/0e10b424b142f7c8c216351df60c4e2a173475f7)